src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -0,0 +1,90 @@
+.va-sidenav {
```

Comment:
This doesn't have to be done now, but if we do migrate this into the design system as its own component, we'll need to update it use the design system's prefix for components like `vads-c-button`